src/Post.js

```diff
@@ -0,0 +1,36 @@
+import { useState } from "react";
+var x=1
+
+function Post(){
+let [count,setcounter] = useState(0)
+if(count>=100){
+    alert(`You Win!`)
+}
+function setval(){
+    if(count<10){
+        alert("You can't afford that!")
+    }
+     else {
+count=count -10;
+x+=1
+    }
+
+    
+}
+
+
+function PlayAgain(){
+    window.location.reload()}
```

Comment:
Ah we would want to restart the game without reloading the page. This'll mean that rather than doing a full refresh, we just update the state values to be what they were at the start.